modules/gapi/include/opencv2/gapi/infer/onnx.hpp

```diff
@@ -634,6 +635,19 @@ template<typename Net> class Params {
         return *this;
     }
 
+    /** @brief Configures session options for ONNX Runtime.
+
+    This function allows setting various session options for the ONNX Runtime
+    session by accepting a map of key-value pairs.
+
+    @param options A map of session options to be applied to the ONNX Runtime session.
+    @return the reference on modified object.
+    */
+    Params<Net>& cfgSessionOptions(const std::unordered_map<std::string, std::string>& options) {
```

Comment:
also should accept just `map` to comply with python bindings

Reply:
Yes